### PR TITLE
[Her-46] Migrate entity structure

### PR DESCRIPTION
**Pull request by TBiannca**
## Related
[HER-46](https://nerds-interns.atlassian.net/browse/HER-46?atlOrigin=eyJpIjoiZGZlNTFmZjFhNWI1NDg3MGFmOTg2MzU5NGRlNDU1OGIiLCJwIjoiaiJ9)

### CODE REVIEW

File: src/Data/Migrations/20210708111637_AlterDetectionsAddFields.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    public partial class AlterDetectionsAddFields : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "Class",
+                table: "Detections",
+                type: "text",
+                nullable: true);
```
Comment (miutamihai) [inline src/Data/Migrations/20210708111637_AlterDetectionsAddFields.cs:14]: Let's make sure that this column is not null able, as no `EDetection` should be created without the `class` property being set.

Comment (miutamihai) [reply src/Data/Migrations/20210708111637_AlterDetectionsAddFields.cs:14]: Read [this](https://docs.microsoft.com/en-us/ef/core/modeling/entity-properties?tabs=data-annotations%2Cwithout-nrt) article on data annotations to figure out how to do it.

**Review by miutamihai — CHANGES_REQUESTED** [review]

**Review by mihairadulescu — DISMISSED** [review]
set nullable to false and close it.

**Review by miutamihai — APPROVED** [review]
LGTM

**Comment by miutamihai** [comment]
@RaduBuciuceanu we tested it and applying the migration whilst the table holds data has no issues.

**Review by RaduBuciuceanu — APPROVED** [review]

**Pull request merged**